.github/workflows/alpine-32bit-build-and-test.yaml

```diff
@@ -74,3 +74,10 @@ jobs:
         name: PostgreSQL log
         path: postmaster.log
 
+    - name: Slack Notification
+      if: failure()
+      env:
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
Should we specify channel here?

Reply:
the channel is configured in the webhook configuration, but it could be overwritten here